modules/statistics/php/stats_demographic.class.inc

```diff
@@ -77,14 +78,14 @@ class Stats_Demographic extends \NDB_Form
         $dropdown_name,
         $dropdown_opt,
         $dropdown_selected,
-        $centres,
+        $centers,
         $data,
         $Subsection="",
         $disclamer='',
         $projectID=null
     ) {
-
-        $tpl_data = array();
+        $projectID = $projectID == '' ? null : $projectID;
+        $tpl_data  = array();
```

Comment:
Since this is a cleanup PR, could you use short array syntax? ie. `$tpl_data = [];`.

I don't know if we have a policy on this exactly but this is a more modern way of doing PHP iirc.

Reply:
@ZainVirani I'm wrong actually. Apparently old array syntax is our standard so leave this as they are.